### PR TITLE
Support rpmdeplint

### PR DESCRIPTION
**Pull request by gsr-shanks**
Signed-off-by: gsr-shanks <gsr@redhat.com>

### CODE REVIEW

**Comment by mikebonnet** [comment]
@gsr-shanks Could you rebase this to remove the merge commit?

File: fedmsg_meta_umb/rpmdeplint.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+log = logging.getLogger(__name__)
+
+
+class RPMDEPLINTProcessor(BaseProcessor):
+    topic_prefix_re = r'/topic/VirtualTopic\.eng'
+
+    __name__ = 'rpmdeplint'
+    __description__ = 'find errors in RPM packages in the context of their dependency graph'
+    __obj__ = 'rpmdeplint'
+    __docs__ = 'https://rpmdeplint.readthedocs.io/en/latest/'
```
Comment (danc86) [inline fedmsg_meta_umb/rpmdeplint.py]: Can you please change this to https://docs.pagure.org/rpmdeplint/ ? I am trying to get away from the readthedocs stuff in favour of consolidating on pagure.

File: fedmsg_meta_umb/tests/test_rpmdeplint.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors: Gowrishankar Rajaiyan <grajaiya@redhat.com>
+
+import fedmsg.tests.test_meta
+
+
+class TestRPMDEPLINTStart(fedmsg.tests.test_meta.Base):
+    """ `RPMDEPLINT <https://rpmdeplint.readthedocs.io/en/latest/`_ finds errors in RPM packages in the context
```
Comment (danc86) [inline fedmsg_meta_umb/tests/test_rpmdeplint.py]: It's not an acronym, I suggest just spelling it "rpmdeplint".

    The `rpmdeplint <https://pagure.io/rpmdeplint>`_ tool finds errors...

Also you have a Sphinx syntax error here (missing `>`).

**Comment by thrix** [comment]
As discussed with @gsr-shanks, seems we should be not extending the meta format here, just use eng.ci topic and the defined messages as others do. Subtests can go to xunit files for now, as there is no special need to waive subresults as we do for RPMdiff for example.

**Comment by ralphbean** [comment]
OK - @thrix, what should we do with this?  Will you guys be moving the rpmdiff messages into the `.ci` hierarchy?

If so, should we merge this to still render old rpmdiff messages in datagrepper for historical queries?

Or, we could drop it just to keep things simple?

**Comment by gsr-shanks** [comment]
Yes, I will be moving the rpmdiff messages into the eng.ci hierarchy. I have already asked for permissions to publish.

We can drop it just to keep things simple. :-) 

If needed, I shall create another PR.

**Comment by ralphbean** [comment]
Sorry to let this sit for so long.  Closing now.

**Pull request closed without merging**